cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/TapChanger.java

```diff
@@ -173,6 +180,11 @@ public TapChanger setTapChangerControlEnabled(boolean tapChangerControlEnabled)
         return this;
     }
 
+    public TapChanger setHiddenCombinedTapChanger(TapChanger hiddenCombinedTapChanger) {
```

Comment:
For hidden combined tap changer: see my comment about the two modes of the conversion. I am not sure we have to support that in the mode ````USING_ONLY_NETWORK_PARAMETER````.